src/components/admin/dashboard-statistics/CountryStats.tsx

```diff
@@ -84,7 +66,7 @@ const CountryStats = () => {
         <PelicoCard message={'Merci de sélectionner un pays pour analyser ses statistiques'} />
       ) : (
         <Box mt={2}>
-          {isLoadingCountryStatisticsForGraphs ? (
+          {isLoadingCountryStatisticsForGraphs || isLoadingVillages ? (
```

Comment:
```suggestion
          {isLoadingCountryStatisticsForGraphs ? (
```
Vu que isLoadingVillages est déjà inclus dans le isLoadingCountryStatisticsForGraphs